docs/getting-started/first-broker.md

```diff
@@ -9,29 +9,30 @@ Let's take a look at how these components interact:
-| `Broker`       | :material-relation-many-to-many: A "hub" for events in your infrastructure; a central location to send events for delivery. |
-| `Trigger`    | :material-magnet-on: Acts as a filter for events entering the broker, can be configured with desired event attributes. |
-| `Sink`    | :material-download:  A destination for events. |
+|:material-information: **Source**       |A Kubernetes Custom Resource which emits events to the Broker. |
+|:material-relation-many-to-many: **Broker**       | A "hub" for events in your infrastructure; a central location to send events for delivery. |
+|:material-magnet: **Trigger** |Acts as a filter for events entering the broker, can be configured with desired event attributes. |
+|:material-download: **Sink** | A destination for events. |
 
-!!! note "A note on `Sources` and `Sinks`"
-    A Knative Service can act as both a `Source` and a `Sink` for events, and for good reason. You may want to consume events from the `Broker` and send modified events back to the `Broker`, as you would in any pipeline use-case.
+!!! note "A note on Sources and Sinks"
+    A Knative Service can act as both a Source and a Sink for events, and for good reason. You may want to consume events from the Broker and send modified events back to the Broker, as you would in any pipeline use-case.
 
 ### CloudEvents
-Knative Eventing uses <a href="https://github.com/cloudevents/spec/blob/master/primer.md" target="blank_">CloudEvents</a> send information back and forth between your Services and these components.
+Knative Eventing uses <a href="https://github.com/cloudevents/spec/blob/master/primer.md" target="blank_">CloudEvents</a> to send information back and forth between your Services and these components.
 
 ??? question "What are CloudEvents?"
     For our purposes, the only thing you need to know about CloudEvents are:
 
+    1. CloudEvents can carry some attributes (like id, Source, type, etc) as well as data payloads (JSON, plaintext, reference to data that lives elsewhere, etc).
+    1. CloudEvents can be "emitted" by almost anything and can be transported to anywhere in your deployment.
     1. CloudEvents follow the <a href = "https://github.com/cloudevents/spec" target="_blank">CloudEvents 1.0 Specification</a>, with required and optional attributes.
-    1. CloudEvents can be "emitted" by almost anything and can be transported to anywhere in your deployment.  
-    1. CloudEvents can carry some attributes (things like `id`, `source`, `type`, etc) as well as data payloads (JSON, plaintext, reference to data that lives elsewhere, etc).
+
 
     To find out more about CloudEvents, check out the [CloudEvents website](https://cloudevents.io/)!
 
 
 ## Examining the Broker
-As part of the `KonK` install, you should have an in-memory `Broker` already installed.
+As part of the `KonK` install, you should have an in-memory Broker already installed.
```

Comment:
As part of the KonK installation, you should have an in-memory Broker already installed.

Reply:
KonK installed the broker, changed it to: 

As part of the `KonK` install, an In-Memory Broker should have already be installed in your Cluster. Check to see that it is installed by running the command: